selfdrive/car/honda/values.py

```diff
@@ -597,7 +600,7 @@ class CAR:
       b'46114-TLA-A040\x00\x00',
       b'46114-TLA-A050\x00\x00',
       b'46114-TLA-A930\x00\x00',
-      b'46114-TMC-U020\x00\x00',
+      b'46114-TMC-U020\x00\x00',      
```

Comment:
Just remove this extra space and we're good to go!

Reply:
Good catch! Extra space removed.